package-lock.json

```diff
@@ -0,0 +1,13698 @@
+{
```

Comment:
Make sure to add`package-lock.json` to `.gitignore`

Reply:
👍 Ideally delete it.